### PR TITLE
Make get_properties optional

### PR DESCRIPTION
**Pull request by danieljanes**

### CODE REVIEW

**Review by tanertopal — DISMISSED** [review]
Looks awesome! Approved. Just one failing linter error.

File: src/py/flwr/client/app.py [hunk 1]
```diff
@@ -160,6 +162,12 @@ class `flwr.client.NumPyClient`.
     # Wrap the NumPyClient
     flower_client = NumPyClientWrapper(client)
 
+    # Remove NumPyClientWrapper.get_properties with the base class version
+    # Client.get_properties if method was not overridden by user-provided
+    # NumPyClient instance
```
Comment (tanertopal) [inline src/py/flwr/client/app.py:167]: Can you elaborate this further or rephrase. I don't think I understand whats happening here.

File: src/py/flwr/client/app.py [hunk 2]
```diff
@@ -207,6 +215,12 @@ class `flwr.client.KerasClient`.
     # Wrap the Keras client
     flower_client = KerasClientWrapper(client)
 
+    # Remove KerasClientWrapper.get_properties with the base class version
+    # Client.get_properties if method was not overridden by user-provided
+    # KerasClient instance
```
Comment (tanertopal) [inline src/py/flwr/client/app.py:220]: Same as above.

File: src/py/flwr/client/app.py (hunk 1 above)
Comment (danieljanes) [reply src/py/flwr/client/app.py]: @tanertopal updated the comment, is it better now?

File: src/py/flwr/client/app.py (hunk 2 above)
Comment (danieljanes) [reply src/py/flwr/client/app.py]: Same as above.

File: src/py/flwr/client/app.py (hunk 1 above)
Comment (tanertopal) [reply src/py/flwr/client/app.py]: Yes, thx!

**Review by tanertopal — APPROVED** [review]

**Pull request merged**